packages/modular-scripts/src/convert.ts

```diff
@@ -164,7 +164,7 @@ export async function convert(cwd: string = process.cwd()): Promise<void> {
     });
 
     logger.log('Validating your modular project...');
-    await check();
+    await check(false, cwd);
```

Comment:
This is a bit difficult to read. Can we change all interfaces to
`{ fix: boolean, targetDirectory?: string }`